tools/genenotebook/find_free_port.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+
+# This script finds a free port on the local machine in the 7000-65000 interval
+# Should work in biocontainers derived from conda
+# Taken from https://unix.stackexchange.com/a/358101
+
+netstat -aln | awk '
```

Comment:
netstat is getting phased out and may not be available (unless it's provided by GNB?)

Reply:
If it's not there, you can use `ss -alnut` (aln are the same, UT for udp+tcp without unix), it'll be column 2 for LISTEN, and 5 for the address.

Reply:
netstats is by default in biocontainers images, but not ss (yet?). As I don't think there's a conda package for that (maybe I'm wrong), to be safe(r) I'm using the first available between ss and netstat

Reply:
Ah ok, good @ biocontainers. That's fine then!